inc/classes/customizer/class-base.php

```diff
@@ -67,12 +67,18 @@ public function page_choices() {
 		$posts = new WP_Query(
 			array(
 				'post_type'      => 'page',
-				'posts_per_page' => 100,
+				'posts_per_page' => -1,
+				'orderby'        => 'title',
+				'order'          => 'ASC',
+				'post_status'    => 'publish',
 			)
 		);
 
 		foreach ( $posts->posts as $post_choice ) {
-			$choices[ $post_choice->ID ] = $post_choice->post_title;
+			if ( ! empty( $post_choice->post_title ) ) {
+				$page_relative_link = str_replace( home_url(), '', get_permalink( $post_choice->ID ) );
```

Comment:
There's a core function [wp_make_link_relative()](https://developer.wordpress.org/reference/functions/wp_make_link_relative/) that might do what you want here. It's not exactly the same, because it doesn't handle sites where the home URL is in a subdirectory, but it's similar.